packages/base/links-to-editor.gts

```diff
@@ -143,6 +147,7 @@ export class LinksToEditor extends GlimmerComponent<Signature> {
 
   private chooseCard = restartableTask(async () => {
     let type = identifyCard(this.args.field.card) ?? baseCardRef;
+    type = await getNarrowestType(this.args.subclassType, type, myLoader());
```

Comment:
Alternatively, you can only call this function if a subclassType arg was provided

Reply:
Actually do you even need to check if the `type` is the base CardDef? We know that all cards descend from it, so maybe in that case you can skip loading the whole chain and just use the provided subtype.

Reply:
> Actually do you even need to check if the type is the base CardDef? We know that all cards descend from it, so maybe in that case you can skip loading the whole chain and just use the provided subtype.

I think we do. We need to check if type is base CardDef as a terminating condition to "stop checking".

Reply:
@burieberry maybe we can talk thru this after standup. Probably some small misunderstanding

Reply:
@burieberry I have made the change that you have suggested